<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="1" tests="2" time="5.573">
 <testsuite name="calculator tests" timestamp="2024-04-17T16:44:08" hostname="localhost" time="5.573" errors="0" tests="2" skipped="0" disabled="0" failures="1">
  <testcase classname="calculator tests" name="addition test" time="2.963">
   <failure type="exception" message="Failed: No element found using locator: by.cssContainingText(&quot;.ng-binding&quot;, &quot;114&quot;)"><![CDATA[NoSuchElementError: No element found using locator: by.cssContainingText(".ng-binding", "114")
    at C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27
    at ManagedPromise.invokeCallback_ (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
    at C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at processTicksAndRejections (node:internal/process/task_queues:95:5)Error
    at ElementArrayFinder.applyAction_ (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.<computed> [as getText] (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.<computed> [as getText] (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at homepage.verifyResult (C:\Users\SujanArpita\Downloads\Protractor_Tutorial\pages\homepage.js:19:23)
    at UserContext.<anonymous> (C:\Users\SujanArpita\Downloads\Protractor_Tutorial\tests\calculator.js:11:18)
    at C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
From: Task: Run it("addition test") in control flow
    at UserContext.<anonymous> (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
From asynchronous test: 
Error
    at Suite.<anonymous> (C:\Users\SujanArpita\Downloads\Protractor_Tutorial\tests\calculator.js:6:5)
    at addSpecsToSuite (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\SujanArpita\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.<anonymous> (C:\Users\SujanArpita\Downloads\Protractor_Tutorial\tests\calculator.js:5:1)
    at Module._compile (node:internal/modules/cjs/loader:1369:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)]]>
   </failure>
  </testcase>
  <testcase classname="calculator tests" name="substraction test" time="2.588" />
 </testsuite>
</testsuites>